src/backend/tests/stats/test_stats_save_result.py

```diff
@@ -0,0 +1,57 @@
+from flask import g
+
+import expungeservice
+from expungeservice.database.db_util import rollback_errors
+from expungeservice.stats import save_result
+from tests.factories.crawler_factory import CrawlerFactory
+from expungeservice.database import get_database
+from tests.endpoints.endpoint_util import EndpointShared
+
+
+class TestStats():
+
+    def test_save_result(self):
+
+        with expungeservice.create_app("development").app_context():
+
+            expungeservice.request.before()
+            database = get_database()
```

Comment:
There is some redundancy in the above two lines: see https://github.com/codeforpdx/recordexpungPDX/blob/master/src/backend/expungeservice/request/request.py#L8